test/functional/wrapper.js

```diff
@@ -139,7 +139,7 @@ describe('Wrapper', () => {
     const autocompleteList = autocompleteInstance.querySelector('.autocomplete__list')
     expect(autocompleteInput.tagName.toLowerCase()).to.equal('input')
     expect(autocompleteInput.id).to.equal(id)
-    expect(autocompleteList.tagName.toLowerCase()).to.equal('ul')
+    expect(autocompleteList).to.equal(null)
```

Comment:
It looks like we're now missing a test for when there _are_ more than zero options 🤔

Reply:
We have [one that captures this](https://github.com/kevindew/accessible-autocomplete/blob/master/test/functional/wrapper.js#L183-L184) already.

Reply:
Ah, great 👍 is it worth adding the line we've removed here, to that test? (Asserting that the containing element is a `<ul>`?)

Reply:
Updated the mentioned test to capture both the element type and the associated class.

Reply:
Looks good 👍